README.md

```diff
@@ -650,34 +757,48 @@ Payments(paymentsContractAddress).modifyRailLockup(
 );
 ```
 
-### 7. Deal Cancellation
+### 7. Ending a Deal
 
-When a user cancels a deal, the service contract will modify the rail's payment to take that into account. In this case, the service contract sets the rail's payment rate to zero and pays a fixed termination fee out of the rail's "fixed" lockup.
+There are two primary ways to end a deal:
+
+**Method 1: Soft End (Rate to Zero)**
+The operator can set the payment rate to zero and optionally charge a final termination fee. This keeps the rail active but stops recurring payments.
 
 ```solidity
-// Service contract reduces payment rate and possibly issues a termination payment
+// Service contract reduces payment rate and issues a termination payment
```

Comment:
above you said "optionally charge a final termination fee" - is that the same thing as "termination payment" and therefore should that optionality be reflected in the language here?

Reply:
Good catch. Done.